codalab/server/bundle_manager.py

```diff
@@ -356,7 +354,9 @@ def get_available_workers(user_id):
                         )
                         # Don't start this bundle yet, as there is no parallel_run_quota left for this user.
                         continue
-            if not workers_list: # Length is 0 (private user with no workers) or is None (root user)
+            if (
+                not workers_list
+            ):  # Length is 0 (private user with no workers) or is None (root user)
```

Comment:
Is this because of black? The formatting looks off.

Reply:
I think black includes the comment in the line length, hence it breaks it. 
I'll move the comment above the condition so it doesn't break.